<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>90</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="PyDMLabel" name="ion_chamber_name_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Ipreslit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="channel" stdset="0">
      <string>sig://${IC}.description</string>
     </property>
     <property name="displayFormat" stdset="0">
      <enum>PyDMLabel::String</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMAnalogIndicator" name="ion_chamber_indicator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="showUnits" stdset="0">
      <bool>true</bool>
     </property>
     <property name="PyDMToolTip" stdset="0">
      <string>The measured voltage output from the pre-amp.</string>
     </property>
     <property name="channel" stdset="0">
      <string>sig://${IC}.voltmeter.volts</string>
     </property>
     <property name="showValue" stdset="0">
      <bool>false</bool>
     </property>
     <property name="showLimits" stdset="0">
      <bool>true</bool>
     </property>
     <property name="showTicks" stdset="0">
      <bool>true</bool>
     </property>
     <property name="invertedAppearance" stdset="0">
      <bool>false</bool>
     </property>
     <property name="limitsFromChannel" stdset="0">
      <bool>false</bool>
     </property>
     <property name="userUpperLimit" stdset="0">
      <double>5.000000000000000</double>
     </property>
     <property name="minorAlarmFromChannel" stdset="0">
      <bool>false</bool>
     </property>
     <property name="userUpperMinorAlarm" stdset="0">
      <double>4.500000000000000</double>
     </property>
     <property name="userLowerMinorAlarm" stdset="0">
      <double>0.500000000000000</double>
     </property>
     <property name="majorAlarmFromChannel" stdset="0">
      <bool>false</bool>
     </property>
     <property name="userUpperMajorAlarm" stdset="0">
      <double>5.000000000000000</double>
     </property>
     <property name="userLowerMajorAlarm" stdset="0">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="volts_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="PyDMLabel" name="volts_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>#.##</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="precision" stdset="0">
          <number>2</number>
         </property>
         <property name="showUnits" stdset="0">
          <bool>false</bool>
         </property>
         <property name="precisionFromPV" stdset="0">
          <bool>false</bool>
         </property>
         <property name="PyDMToolTip" stdset="0">
          <string>The measured voltage output from the pre-amp.</string>
         </property>
         <property name="channel" stdset="0">
          <string>sig://${IC}.voltmeter.volts</string>
         </property>
         <property name="enableRichText" stdset="0">
          <bool>false</bool>
         </property>
         <property name="displayFormat" stdset="0">
          <enum>PyDMLabel::Decimal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="volts_unit_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string> V </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="current_layout">
       <property name="topMargin">
        <number>4</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="PyDMLabel" name="current_value_label">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>#.##e##</string>
         </property>
         <property name="showUnits" stdset="0">
          <bool>false</bool>
         </property>
         <property name="PyDMToolTip" stdset="0">
          <string>The calculated current based on the voltage from the voltmeter and the pre-amp settings.</string>
         </property>
         <property name="channel" stdset="0">
          <string>sig://${IC}.voltmeter.amps</string>
         </property>
         <property name="displayFormat" stdset="0">
          <enum>PyDMLabel::Exponential</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="current_unit_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>A</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="lytVoltmeterGainBox">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="gain_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Gain/Offset</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lytVoltmeterGainButtons">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="PyDMPushButton" name="gain_down_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>←</string>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
         <property name="rules" stdset="0">
          <string>[{&quot;name&quot;: &quot;Gain button lower range&quot;, &quot;property&quot;: &quot;Enable&quot;, &quot;initial_value&quot;: &quot;True&quot;, &quot;expression&quot;: &quot;ch[0] &lt; 27&quot;, &quot;channels&quot;: [{&quot;channel&quot;: &quot;sig://${IC}.preamp.gain_level&quot;, &quot;trigger&quot;: true, &quot;use_enum&quot;: false}], &quot;notes&quot;: &quot;If the sensitivity is at its highest setting, disable the button so users don't click it unnecessarily.&quot;}]</string>
         </property>
         <property name="PyDMToolTip" stdset="0">
          <string>Decrease the pre-amp gain (higher sensitivity values)</string>
         </property>
         <property name="channel" stdset="0">
          <string>sig://${IC}.preamp.gain_level</string>
         </property>
         <property name="confirmMessage" stdset="0">
          <string>Are you sure you want to proceed?</string>
         </property>
         <property name="pressValue" stdset="0">
          <string>1</string>
         </property>
         <property name="relativeChange" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PyDMPushButton" name="gain_up_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>→</string>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
         <property name="rules" stdset="0">
          <string>[{&quot;name&quot;: &quot;Gain button upper range&quot;, &quot;property&quot;: &quot;Enable&quot;, &quot;initial_value&quot;: &quot;True&quot;, &quot;expression&quot;: &quot;ch[0] &gt; 0&quot;, &quot;channels&quot;: [{&quot;channel&quot;: &quot;sig://${IC}.preamp.gain_level&quot;, &quot;trigger&quot;: true, &quot;use_enum&quot;: false}], &quot;notes&quot;: &quot;&quot;}]</string>
         </property>
         <property name="alarmSensitiveContent" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alarmSensitiveBorder" stdset="0">
          <bool>false</bool>
         </property>
         <property name="PyDMToolTip" stdset="0">
          <string>Increase the pre-amp gain (lower sensitivity values)</string>
         </property>
         <property name="monitorDisp" stdset="0">
          <bool>false</bool>
         </property>
         <property name="channel" stdset="0">
          <string>sig://${IC}.preamp.gain_level</string>
         </property>
         <property name="passwordProtected" stdset="0">
          <bool>false</bool>
         </property>
         <property name="password" stdset="0">
          <string/>
         </property>
         <property name="protectedPassword" stdset="0">
          <string/>
         </property>
         <property name="showConfirmDialog" stdset="0">
          <bool>false</bool>
         </property>
         <property name="confirmMessage" stdset="0">
          <string>Are you sure you want to proceed?</string>
         </property>
         <property name="pressValue" stdset="0">
          <string>-1</string>
         </property>
         <property name="releaseValue" stdset="0">
          <string>None</string>
         </property>
         <property name="relativeChange" stdset="0">
          <bool>true</bool>
         </property>
         <property name="writeWhenRelease" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="PyDMLabel" name="sens_num_label">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>## GA/V</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="channel" stdset="0">
          <string>sig://${IC}.preamp.sensitivity_text</string>
         </property>
         <property name="displayFormat" stdset="0">
          <enum>PyDMLabel::String</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Include this ion chamber when using the adaptive plan for setting gains.</string>
       </property>
       <property name="text">
        <string>Auto-gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settings_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Open a new window with settings specific to this ion chamber.</string>
       </property>
       <property name="text">
        <string>More</string>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
  <customwidget>
   <class>PyDMScaleIndicator</class>
   <extends>QFrame</extends>
   <header>pydm.widgets.scale</header>
  </customwidget>
  <customwidget>
   <class>PyDMAnalogIndicator</class>
   <extends>PyDMScaleIndicator</extends>
   <header>pydm.widgets.analog_indicator</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
